From: Roger Shimizu <rogershimizu@gmail.com>
Date: Sun, 24 Sep 2017 11:55:15 +0900
Subject: Update mirror list
Forwarded: https://github.com/micahflee/torbrowser-launcher/pull/289

Except the official site, there're only 3 working mirror in current
mirror list. So it's really necessary to update the list now.

Got the latest list from:
 - https://www.torproject.org/getinvolved/mirrors.html.en
And only keeps https links for security sake.
---
 share/torbrowser-launcher/mirrors.txt | 42 +++++++++++++++++++----------------
 1 file changed, 23 insertions(+), 19 deletions(-)

diff --git a/share/torbrowser-launcher/mirrors.txt b/share/torbrowser-launcher/mirrors.txt
index cc24c7a..529b0bf 100644
--- a/share/torbrowser-launcher/mirrors.txt
+++ b/share/torbrowser-launcher/mirrors.txt
@@ -1,23 +1,27 @@
 https://www.torproject.org/dist/
-https://www.torservers.net/mirrors/torproject.org/dist/
-https://mirror.ml/tor/dist/
-https://tor.spline.inf.fu-berlin.de/dist/
-https://tormirror.almnet.de/dist/
-https://tor.dev-random.de/dist/
-https://www.unicorncloud.org/public/torproject.org/dist
-https://www.oignon.net/dist/
-https://mirror.hackthissite.org/tor
-https://tor.linuxlounge.net/dist/
-https://torproject.cryptowars.info/dist/
-https://tor.crazyhaze.de/dist/
+https://mirror.unicorncloud.org/torproject.org/dist/
 https://creep.im/tor/dist/
-https://mirror.torland.me/torproject.org/dist/
+https://mirror.velcommuta.de/tor/dist/
+https://reichster.de/mirrors/torproject.org/dist/
+https://www.moparisthebest.com/tor/dist/
+https://sela.io/mirrors/torproject.org/dist/
+https://nl.mirror.babylon.network/torproject/dist/
+https://fr.mirror.babylon.network/torproject/dist/
+https://fbnaia.homelinux.net/torproject/dist/
 https://tor.myrl.net/dist/
-https://torprojekt.userzap.de/dist/
-https://tor.myrl.net/dist/
-https://tor.beme-it.de/dist/
-http://tor.borgmann.tv/dist/
-https://otivpn.com/tor/dist
-https://www.eprci.com/tor/dist/
+https://tor.eprci.net/dist/
 https://tor.stalkr.net/dist/
-https://torproject.gtor.org/dist/
+https://tormirror.tb-itf-tor.de/dist/
+https://tor.zilog.es/dist/
+https://torproject.ph3x.at/dist/
+https://tor-mirror.cyberguerrilla.org/dist/
+https://tor.fr33tux.org/dist/
+https://tor.ludikovsky.name/dist/
+https://tor.ybti.net/dist/
+https://tor.0x3d.lu/dist/
+https://torproject.urown.net/dist/
+https://tor.armbrust.me/dist/
+https://torproject.mirror.metalgamer.eu/dist/
+https://tor.heikorichter.name/dist/
+https://tor.eff.org/dist/
+https://tor.void.gr/dist/
